geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B F(1)   1.3239(2)
+B F(2)   1.3139(2)
+B O      1.3426(3)
+O H      0.9581(6)
+O B F(1) 122.31(2)
+B O H    113.09(4)
+O B F(2) 119.47(2)
+F B F    118.22(3)
+
+NAME = BH2
+# Point Group: C2v
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Sunahori FX, Gharaibeh M, Clouthier DJ, Tarroni R (2015) BH2 revisited: New, extensive measurements of
+laser-induced fluorescence transitions and ab initio calculations of near-spectroscopic accuracy. J Chem Phys
+142(17):174302/1-174302/13
+[http://dx.doi.org/10.1063/1.4919094]
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B H   1.197(2)
+H B H 129.6(2)
+
+NAME = BH6N
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Trigonal pyramidal
+# Bond lengths: rm^(1)     (angstroms)
+# Bond angles:  thetam^(1) (degrees)
+# Source Paper: Demaison J, Lievin J, Császár AG, Gutle C (2008) Equilibrium structure and torsional barrier of BH3NH3. J
+Phys Chem A 112(19):4477-4482
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B...N   1.6576(16)
+B   H   1.2098(2)
+N   H   1.0083(3)
+N...B H 104.73(1)
+B...N H 110.29(2)
+
+# Bond lengths: re^se     (angstroms)
+# Bond angles:  thetae^se (degrees)
+
+B...N   1.6453(1)
+B   H   1.2058(3)
+N   H   1.0101(3)
+N...B H 105.00(2)
+B...N H 110.97(2)
+
+NAME = BeI2
+# Point Group: Dooh
+# Shape: Bent
+# Bond lengths: rg     (angstroms)
+# Bond angles:  thetag (degrees)
+# Source Paper: Shlykov SA, Zhabanov YA, Giricheva NI, Girichev AG, Girichev GV (2015) Combined gas electron
+diffraction/mass spectrometric study of beryllium diiodide assisted by quantum chemical calculations: structure
+and thermodynamics of beryllium dihalides. Struct Chem 26 (5-6):1451-1458
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+# Temperature = 501(5) Kelvin
+
+Be I   2.163(6)
+I  Be I 165(3)
+
+# Temperature = 722(10) Kelvin
+
+Be I   2.172(7)
+I  Be I 162(4)
+
+NAME = BrGeH
+# Point Group: Cs
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Kang L, Sunahori F, Minei AJ, Clouthier DJ, Novick SE (2009) Fourier transform microwave spectroscopy of
+monobromogermylene (HGeBr and DGeBr), a heavy atom carbene analog. J Chem Phys 130(12):124317/1-
+124317/11
+doi: 10.1063/1.3080161
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Ge Br   2.3276(9)
+Ge H    1.6093(7)
+H Ge Br 92.44(4)
+
+# Bond lengths: rm(1)     (angstroms)
+# Bond angles:  thetam(1) (degrees)
+
+Ge Br   2.3243(2)
+Ge H    1.563(9)
+H Ge Br 93.55(24)
+
+NAME = BrHO 2010
+# Point Group: Cs
+# Shape: Bent
+# Bond lengths: re     (angstroms)
+# Bond angles:  thetae (degrees)
+# Source Paper: Cohen EA, Müller HSP, Tan TL, McRae GA (2010) High resolution spectroscopy of DOBr and molecular
+properties of hypobromous acid. J Mol Spectrosc 262(1):30-36
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+H O    0.9640(3)
+O Br   1.82800(1)
+H O Br 102.99(1)
```

Comment:
Lining up the decimals would be nice. Rather than writing this every time, perhaps your next commit can have this fixed for the rest of the file.